.github/workflows/test.yml

```diff
@@ -25,6 +25,8 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v2
+    - name: Install build dependencies
+      run: sudo dnf -y install openssl
```

Comment:
`run_tests.sh` installs other build dependencies. It is probably better to add it there: https://github.com/keylime/keylime/blob/master/test/run_tests.sh